core/src/main/scala/cromwell/util/JsonEditor.scala

```diff
@@ -169,23 +206,75 @@ object JsonEditor {
     doUpdateWorkflow(workflowJson = json)
   }
 
+  private def callsObject(workflowObject: JsonObject): ErrorOr[Option[JsonObject]] = {
+    workflowObject("calls") match {
+      case None => None.validNel
+      case Some(callsJson) =>
+        callsJson.asObject match {
+          case None => s"'calls' member unexpectedly not a JSON object: $callsJson".invalidNel
+          case Some(calls) => Option(calls).validNel
+        }
+    }
+  }
+
+  private def callsObject(workflowJson: Json): ErrorOr[Option[JsonObject]] = {
+    workflowJson.asObject match {
+      case None => s"Workflow JSON unexpectedly not an object: $workflowJson".invalidNel
+      case Some(obj) => callsObject(obj)
+    }
+  }
+
   /**
-    * Update workflow json, replacing "subWorkflowMetadata" elements of root workflow's "calls" object by "subworkflowId"
+    * Update workflow json, replacing "subWorkflowMetadata" elements of root workflow's "calls" object by "subWorkflowId"
     *
     * @param workflowJson json blob
     * @return updated json
     */
-  def replaceSubworkflowMetadataWithId(workflowJson: Json): ErrorOr[Json] = {
-    def replaceSubworkflowMetadataObjectWithSubworkflowIdInCalls(callObject: JsonObject, subworkflowJson: Json): Option[ErrorOr[Json]] = {
-      Option(
+  def unexpandSubworkflows(workflowJson: Json): ErrorOr[Json] = {
+    import common.validation.ErrorOr._
```

Comment:
Isn't this already imported at the top of the file? https://github.com/broadinstitute/cromwell/pull/5303/files#diff-13da7a3f29d249a3d401fe8e84e24b4fR11